client/src/components/ElementEditor/Element.js

```diff
@@ -20,9 +20,14 @@ class Element extends Component {
   /**
    * Take the user to the GridFieldDetailForm to edit the record
    */
-  handleClick() {
+  handleClick(event) {
     const { link } = this.props;
 
+    // TODO Replace with something more unique than a class name, perhaps add an ID?
+    if (event.target.className === 'action-menu__toggle btn btn--no-text btn--icon-xl font-icon-dot-3 btn btn-secondary') {
```

Comment:
To start with you could check whether the target's [classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) contains `action-menu__toggle`

Reply:
I think we did something similar with the Delete button - you should maybe check that. I think it has some code in the click handler for the delete button that stops propagation so the click event doesn't bubble to the parent component as well

Reply:
I think the issue here is that we can't easily alter the click handler for the toggle button. That's taken care of by the ActionMenu component as a part of admin. Perhaps that's something to add to silverstripe/silverstripe-admin#610

Reply:
I see. Maybe the click handler needs a callback prop you can pass in e.g. `beforeHandleClick` which is given the event argument so we can tell it to stop propagation